.storybook/preview.js

```diff
@@ -1,9 +1,9 @@
-import React from "react";
-import { addDecorator } from "@storybook/react";
-import ThemeProvider from "../src/ThemeProvider/ThemeProvider";
-
-addDecorator(story => <ThemeProvider>{story()}</ThemeProvider>);
```

Comment:
We have lost the theme provider which means all of the components in storybook are showing up in default MUI colors rather than the IPG dark blue etc.